<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>980</width>
    <height>547</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Aplikasi Bengkel Telang</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="toolTipDuration">
   <number>-1</number>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>0</y>
      <width>1021</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Narkisim</family>
      <pointsize>23</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>BENGKEL MOTOR TELANG</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>34</x>
      <y>150</y>
      <width>421</width>
      <height>292</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="3" column="1">
      <widget class="QRadioButton" name="radioButton">
       <property name="text">
        <string>Servis Ringan  35.000,-</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Jasa:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_1">
       <property name="text">
        <string>Nama Pemilik:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Jenis Kendaraan:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>No. Plat:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="nama">
       <property name="font">
        <font>
         <kerning>false</kerning>
        </font>
       </property>
       <property name="frame">
        <bool>false</bool>
       </property>
       <property name="cursorPosition">
        <number>0</number>
       </property>
       <property name="placeholderText">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="jeniskendaraan">
       <property name="font">
        <font>
         <kerning>false</kerning>
        </font>
       </property>
       <property name="frame">
        <bool>false</bool>
       </property>
       <property name="placeholderText">
        <string notr="true"/>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QRadioButton" name="radioButton_3">
       <property name="text">
        <string>Servis Berat 60.000,-</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QRadioButton" name="radioButton_2">
       <property name="text">
        <string>Servis Sedang  45.000,-</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="plat">
       <property name="font">
        <font>
         <kerning>false</kerning>
        </font>
       </property>
       <property name="frame">
        <bool>false</bool>
       </property>
       <property name="placeholderText">
        <string/>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="lineEdit_Jasa">
       <property name="frame">
        <bool>false</bool>
       </property>
       <property name="dragEnabled">
        <bool>false</bool>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="placeholderText">
        <string/>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Spareparts:</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QComboBox" name="spareparts">
       <property name="font">
        <font>
         <kerning>false</kerning>
        </font>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="maxCount">
        <number>2147483640</number>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="frame">
        <bool>true</bool>
       </property>
       <item>
        <property name="text">
         <string>----------------------------  pilih sparepart  ----------------------------</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Ban Luar Depan  35.000,-</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Ban Luar Belakang  45.000,-</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Ban Dalam Depan  20.000,-</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Ban Dalam Belakang 20.000,-</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Kampas &amp; Tali Rem Depan  35.000,-</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Kampas &amp; Tali Rem Belakang 40.000,-</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Busi  10.000,-</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Oli Inject  45.000,-</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Oli Carbon  30.000,-</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Oli Gardan  15.000,-</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Oli Mesin  15.000,-</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Lampu Depan  65.000,-</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Lampu Belakang  40.000,-</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Lampu Sein  25.000,-</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Karburator  35.000,-</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Rantai/Belt  80.000,-</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Aki  150.000,-</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QLineEdit" name="lineEdit">
       <property name="font">
        <font>
         <kerning>false</kerning>
        </font>
       </property>
       <property name="frame">
        <bool>false</bool>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="placeholderText">
        <string/>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QComboBox" name="member">
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string>Tidak</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Ya  (diskon 5%)</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Member?</string>
       </property>
      </widget>
     </item>
     <item row="10" column="1">
      <widget class="QLineEdit" name="total">
       <property name="font">
        <font>
         <kerning>false</kerning>
        </font>
       </property>
       <property name="frame">
        <bool>false</bool>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="placeholderText">
        <string/>
       </property>
      </widget>
     </item>
     <item row="10" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Total Bayar:</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="btnreset">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>470</y>
      <width>83</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string> Reset</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>reset.png</normaloff>reset.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>18</width>
      <height>18</height>
     </size>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnsinput">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>470</y>
      <width>83</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>  Simpan</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>save.png</normaloff>save.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>12</width>
      <height>16</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>150</y>
      <width>411</width>
      <height>261</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="gridStyle">
     <enum>Qt::SolidLine</enum>
    </property>
   </widget>
   <widget class="QLabel" name="lblstats2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>470</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Cari">
    <property name="geometry">
     <rect>
      <x>579</x>
      <y>420</y>
      <width>361</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
    <property name="cursorPosition">
     <number>0</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
    <property name="placeholderText">
     <string>Cari data disini</string>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>120</y>
      <width>20</width>
      <height>381</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>120</y>
      <width>431</width>
      <height>341</height>
     </rect>
    </property>
    <property name="title">
     <string>Data Transaksi</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblstats">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>470</y>
      <width>301</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>451</width>
      <height>341</height>
     </rect>
    </property>
    <property name="title">
     <string>Identitas dan Jenis Transaksi</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btndelete">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>470</y>
      <width>83</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>  Hapus</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>delete.png</normaloff>delete.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>13</width>
      <height>13</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>50</y>
      <width>1011</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>0</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>logo.png</normaloff>logo.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>66</width>
      <height>66</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>410</y>
      <width>411</width>
      <height>41</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>410</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>searce.png</normaloff>searce.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>39</width>
      <height>33</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <zorder>groupBox_2</zorder>
   <zorder>label_4</zorder>
   <zorder>label_9</zorder>
   <zorder>groupBox</zorder>
   <zorder>label</zorder>
   <zorder>gridLayoutWidget</zorder>
   <zorder>btnreset</zorder>
   <zorder>btnsinput</zorder>
   <zorder>tableView</zorder>
   <zorder>lblstats2</zorder>
   <zorder>line</zorder>
   <zorder>lblstats</zorder>
   <zorder>btndelete</zorder>
   <zorder>pushButton</zorder>
   <zorder>pushButton_2</zorder>
   <zorder>lineEdit_Cari</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>980</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionRiwayat_Transaksi">
   <property name="text">
    <string>Riwayat Transaksi</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
